mfthomps/CyberCIEGE-port-to-Unity3d
# Issue/278 assets software tabs

updated Software and Assets tab in the Component settings to make it more clear that the software or assets are being added to the computer.

To test:

Run the game with computers that can have software installed and assets assigned. Navigate to the Component tab and verify the UI looks as expected.

Closes #278
